python-frontend/src/test/java/org/sonar/python/semantic/v2/TypeInferenceV2Test.java

```diff
@@ -562,10 +631,13 @@ class B:
         test = "hi"
       class C(A, B):
         pass
-      C.test
```

Comment:
why do you have to remove the access to the field `test` everywhere?

Reply:
discussed